spark/pom.xml

```diff
@@ -668,6 +668,16 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.7</id>
```

Comment:
About adding profile for hadoop-2.7, it was discussed in #158 and decided not to have separate profile.
I'm okay with both changing / keeping decision. I'd like to ask @randerzander @jongyoul @fantazic @felixcheung opinions.